pkg/daemon/controller/controller_test.go

```diff
@@ -222,6 +222,109 @@ func TestReconcileService(t *testing.T) {
 	}
 }
 
+func TestBootstrap(t *testing.T) {
+	// TODO: Once resolvePowershellVariables() is implemented, harden example services with Powershell vars to replace
+	testIO := []struct {
+		name                         string
+		configMapServices            []servicescm.Service
+		expectedServicesNameCmdPairs map[string]string
+		expectErr                    bool
+	}{
+		{
+			name:                         "No services",
+			configMapServices:            []servicescm.Service{},
+			expectedServicesNameCmdPairs: map[string]string{},
+			expectErr:                    false,
+		},
+		{
+			name: "Single bootstrap service",
+			configMapServices: []servicescm.Service{
+				{
+					Name:         "test1",
+					Command:      "test1 --var=TEST",
+					Dependencies: nil,
+					Bootstrap:    true,
+					Priority:     0,
+				},
+			},
+			expectedServicesNameCmdPairs: map[string]string{"test1": "test1 --var=TEST"},
+			expectErr:                    false,
+		},
+		{
+			name: "No bootstrap services",
+			configMapServices: []servicescm.Service{
+				{
+					Name:         "test1",
+					Command:      "test1 arg1",
+					Dependencies: nil,
+					Bootstrap:    false,
+					Priority:     0,
+				},
+				{
+					Name:         "test2",
+					Command:      "test2 arg1 arg2",
+					Dependencies: nil,
+					Bootstrap:    false,
+					Priority:     1,
+				},
+			},
+			expectedServicesNameCmdPairs: map[string]string{},
+			expectErr:                    false,
+		},
+		{
+			name: "Multiple mixed services",
+			configMapServices: []servicescm.Service{
+				{
+					Name:         "test1",
+					Command:      "test1 arg1",
+					Dependencies: nil,
+					Bootstrap:    true,
+					Priority:     0,
+				},
+				{
+					Name:         "test2",
+					Command:      "test2",
+					Dependencies: nil,
+					Bootstrap:    true,
+					Priority:     1,
+				},
+				{
+					Name:         "test3",
+					Command:      "test3 arg1 arg2",
+					Dependencies: nil,
+					Bootstrap:    false,
+					Priority:     2,
+				},
+			},
+			expectedServicesNameCmdPairs: map[string]string{"test1": "test1 arg1", "test2": "test2"},
+			expectErr:                    false,
+		},
+	}
+	for _, test := range testIO {
+		t.Run(test.name, func(t *testing.T) {
+			desiredVersion := "testversion"
+			cm, err := servicescm.GenerateWithData(servicescm.NamePrefix+desiredVersion,
+				"openshift-windows-machine-config-operator", &test.configMapServices, &[]servicescm.FileInfo{})
+			require.NoError(t, err)
+			data, err := servicescm.Parse(cm.Data)
+			require.NoError(t, err)
+			encodedData, err := data.MarshallAndEncode()
+			require.NoError(t, err)
+
+			winSvcMgr := winsvc.NewTestMgr(make(map[string]*winsvc.FakeService))
+			err = RunBootstrap(winSvcMgr, encodedData)
+			if test.expectErr {
+				assert.Error(t, err)
+				return
+			}
```

Comment:
I dont see any expected failure cases in the test list.
This, along with the `expectErr` in the test struct should be removed in this case.